SCons/Tool/MSCommon/vc.py

```diff
@@ -231,8 +231,17 @@ def get_host_target(env):
 
 # if using vswhere, a further mapping is needed
```

Comment:
Minor: maybe update the comment?  Perhaps
```
# if using vswhere, map the command lines to probe for installed VC editions
```

Reply:
How about:
```
# if using vswhere, configure command line arguments to probe for installed VC editions
```

Reply:
fine by me